test/e2e/e2e.go

```diff
@@ -317,16 +317,16 @@ func setupSuite() {
 
 // logClusterImageSources writes out cluster image sources.
 func logClusterImageSources() {
-	masterImg, nodeImg, err := lookupClusterImageSources()
+	controlPlaneImg, nodeImg, err := lookupClusterImageSources()
 	if err != nil {
 		framework.Logf("Cluster image sources lookup failed: %v\n", err)
 		return
 	}
-	framework.Logf("cluster-master-image: %s", masterImg)
+	framework.Logf("cluster-control-plane-image: %s", controlPlaneImg)
 	framework.Logf("cluster-node-image: %s", nodeImg)
 
 	images := map[string]string{
-		"master_os_image": masterImg,
+		"control_plane_os_image": contorlPlaneImg,
 		"node_os_image":   nodeImg,
```

Comment:
I know it's not part of the change necessarily. But maybe rename this to `dataPlaneNodeImg` for consistency and clarification of what type of node this actually belongs to. Wdyt? If you rename it make sure to change the variable initilization too.

Reply:
That sounds like a good change to keep clarification.